api/src/main/resources/workbench.yaml

```diff
@@ -3750,6 +3750,10 @@ definitions:
         type: boolean
       lgbtqIdentity:
         type: string
+      gender:
```

Comment:
This confused me because `gender` is singular: I'd probably name this `selfReportedGenders` or something. Here and in other code.